projects/packages/partner/phpunit.xml.dist

```diff
@@ -9,9 +9,4 @@
 			<directory prefix="test" suffix=".php">tests/php</directory>
 		</testsuite>
 	</testsuites>
-	<filter>
-		<whitelist processUncoveredFilesFromWhitelist="false">
-			<directory suffix=".php">src</directory>
-		</whitelist>
-	</filter>
```

Comment:
It's because it's deprecated 😄  When you run PHPUnit you'll get the following warning:

```
Warning:       Your XML configuration validates against a deprecated schema.
Suggestion:    Migrate your XML configuration using "--migrate-configuration"!
```

Technically we could convert it to `<coverage>` instead I think it's unnecessary 😅 
That being said; I'm not a PHPUnit configuration maistro - more of a general user  - so there might be something I'm missing.
I think we're covered by the `<directory prefix="test" suffix=".php">tests/php</directory>` definition (only `test` prefixes in PHP files, in the `tests/php` directory)

```xml
<!--
<filter>
    <whitelist processUncoveredFilesFromWhitelist="false">
        <directory suffix=".php">src</directory>
    </whitelist>
</filter>
-->
<coverage processUncoveredFiles="false">
    <include>
        <directory suffix=".php">src</directory>
    </include>
</coverage>
```